app/src/main/java/ch/epfl/sweng/studyup/utils/adapters/QuestListViewAdapterStudent.java

```diff
@@ -0,0 +1,54 @@
+package ch.epfl.sweng.studyup.utils.adapters;
```

Comment:
This file is not used, I think it was a conflict but git chose to keep the file just in case, but we should delete it.